pkg/uploader/provider/restic.go

```diff
@@ -198,17 +198,17 @@ func (rp *resticProvider) RunRestore(
 	volumePath string,
 	volMode uploader.PersistentVolumeMode,
 	uploaderCfg map[string]string,
-	updater uploader.ProgressUpdater) error {
+	updater uploader.ProgressUpdater) (int64, error) {
```

Comment:
Why is the int64 returned always 0?

Reply:
Restic path doesn't support to return the total size at present. Since Restic path is being deprecated, we don't want to make any new addition to it.
It doesn't make any impact as Restic path won't be used by data mover and only data mover relies on the total size from backup/restore result.

Reply:
ahh so this is just conforming to an interface. got it.